test/test_tensor_creation_ops.py

```diff
@@ -2121,6 +2121,11 @@ def test_constructor_dtypes(self, device):
         self.assertRaises(TypeError, lambda: torch.set_default_tensor_type('torch.IntTensor'))
         self.assertRaises(TypeError, lambda: torch.set_default_dtype(torch.int64))
 
+        # don't support complex.
+        self.assertRaises(TypeError, lambda: torch.set_default_dtype(torch.complex32))
```

Comment:
I think we're about to remove `complex32`, anyway, so we don't need to worry about testing it

Reply:
we are only removing it in the release so we should still add it